index.html

```diff
@@ -165,65 +164,114 @@ <h2>Multi-Post Stories</h2>
-              <li class="globe-li d-flex">Html</li>
-              <li class="globe-li d-flex">css</li>
-            </ul>
           </div>
-        </div>
-        <div class="portfolio-lang">
-          <div class="portfolio-globe d-flex d-flex-col">
-            <div class="globe-img d-flex">
-              <img class="globe-image" src="images/Icon-frameworks.svg" alt="globe-img" />
+          <div class="portfolio-lang">
+            <div class="portfolio-globe d-flex d-flex-col">
+              <div class="globe-img d-flex">
+                <img class="globe-image" src="images/Icon-frameworks.svg" alt="globe-img" />
+              </div>
+              <h3>Frameworks</h3>
+              <ul class="globe-ul d-flex">
+                <li class="globe-li d-flex">Bootstrap</li>
+                <li class="globe-li d-flex">Ruby on rails</li>
+                <li class="globe-li d-flex">RSpec</li>
+                <li class="globe-li d-flex">Capybara</li>
+                <li class="globe-li d-flex">Selenium</li>
+              </ul>
             </div>
-            <h3>Frameworks</h3>
-            <ul class="globe-ul d-flex">
-              <li class="globe-li d-flex">Bootstrap</li>
-              <li class="globe-li d-flex">Ruby on rails</li>
-              <li class="globe-li d-flex">RSpec</li>
-              <li class="globe-li d-flex">Capybara</li>
-              <li class="globe-li d-flex">Selenium</li>
-            </ul>
           </div>
-        </div>
-        <div class="portfolio-lang">
-          <div class="portfolio-globe d-flex d-flex-col">
-            <div class="globe-img d-flex">
-              <img class="globe-image" src="images/Icon-skill.svg" alt="" />
+          <div class="portfolio-lang">
+            <div class="portfolio-globe d-flex d-flex-col">
+              <div class="globe-img d-flex">
+                <img class="globe-image" src="images/Icon-skill.svg" alt="" />
+              </div>
+              <h3>Skills</h3>
+              <ul class="globe-ul d-flex">
+                <li class="globe-li d-flex">Codekit</li>
+                <li class="globe-li d-flex">Github</li>
+                <li class="globe-li d-flex">Codepen</li>
+                <li class="globe-li d-flex">Gitlab</li>
+                <li class="globe-li d-flex">Terminal</li>
+              </ul>
             </div>
-            <h3>Skills</h3>
-            <ul class="globe-ul d-flex">
-              <li class="globe-li d-flex">Codekit</li>
-              <li class="globe-li d-flex">Github</li>
-              <li class="globe-li d-flex">Codepen</li>
-              <li class="globe-li d-flex">Gitlab</li>
-              <li class="globe-li d-flex">Terminal</li>
-            </ul>
           </div>
         </div>
+      </section>
+      <section class="work">
+        <h3 class="duty">I'm always interested in hearing about new projects, so if you'd like to chat please get in
+          touch.
+        </h3>
+        <form class="form-page">
```

Comment:
[ OPTIONAL ]Please make sure that this form has validation checks for all the fields and use `Formspree` to submit it. Please refer to the form validation milestone for the more detailed set of requirements
![form](https://user-images.githubusercontent.com/35504149/135603718-419c9bab-66f7-4aa2-9d48-1bc1392e1c1b.png)
.